packages/datadog-instrumentations/src/helpers/register.js

```diff
@@ -25,20 +25,15 @@ const names = Object.keys(hooks)
 const pathSepExpr = new RegExp(`\\${path.sep}`, 'g')
 
 const disabledInstrumentations = new Set(
-  DD_TRACE_DISABLED_INSTRUMENTATIONS?.split(',').map(name => normalizePluginEnvName(name, true)) ?? []
+  DD_TRACE_DISABLED_INSTRUMENTATIONS?.split(',')
 )
-const reenabledInstrumentations = new Set()
 
 // Check for DD_TRACE_<INTEGRATION>_ENABLED environment variables
 for (const [key, value] of Object.entries(envs)) {
   const match = key.match(/^DD_TRACE_(.+)_ENABLED$/)
-  if (match && value) {
-    const integration = normalizePluginEnvName(match[1], true)
-    if (isFalse(value)) {
-      disabledInstrumentations.add(integration)
-    } else if (isTrue(value)) {
-      reenabledInstrumentations.add(integration)
-    }
+  if (match && isFalse(value)) {
+    const integration = match[1].toLowerCase()
+    disabledInstrumentations.add(integration)
```

Comment:
I believe we should actually also handle this part different as well.
Until then, keeping the normalization here seems better than not doing that (otherwise it would not be possible to match a couple of integrations).

```suggestion
    disabledInstrumentations.add(normalizePluginEnvName(integration))
```

Reply:
@BridgeAR Any other concerns?